ddtrace/internal/schema/__init__.py

```diff
@@ -0,0 +1,16 @@
+from ddtrace import config
+
+from .span_attribute_schema import SPAN_ATTRIBUTE_SCHEMA_VERSIONS
+
+
+# Span attribute schema
+
+__all__ = ["span_attribute_schema"]
+span_attribute_schema = None
+
+__schema_version = config.trace_span_attribute_schema or "v0"
+assert (
+    __schema_version in SPAN_ATTRIBUTE_SCHEMA_VERSIONS
+), f"Invalid span attribute schema version {__schema_version}.  Options are {SCHEMA_VERSIONS}"
```

Comment:
Under what circumstances could you imagine we fail this assert? Should we make the message more helpful for users to fix this?

Reply:
Updated with longer message